zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/InterpreterFactoryTest.java

```diff
@@ -170,81 +170,6 @@ public void testRemoteRepl() throws Exception {
     assertEquals("value_2", remoteInterpreter.getProperty("property_2"));
   }
 
-  /**
```

Comment:
This would have caused CI to fail. 
Let me revert this, and instead of removing it, let me comment it out.